File: C#/24 Hours Clock Time.cs
using System;

namespace _24_Hours_Clock_Time;

public class Solution
{
    static void Main()
    {
        string Line1 = Console.ReadLine() ?? "";
        DateTime Date = new(2023, 12, 12, int.Parse(Line1.Split(':')[0]), int.Parse(Line1.Split(':')[1]), 0);
        int Min = int.Parse(Console.ReadLine());
        Date = Date.AddMinutes(Min);
        Console.WriteLine((Date.Hour < 10 ? "0" + Date.Hour : Date.Hour) + ":" + (Date.Minute < 10 ? "0" + Date.Minute : Date.Minute));
    }
}

File: Python/24 Hours clock time.py
import datetime

line1 = input()
time_parts = line1.split(':')
date = datetime.datetime(2023, 12, 12, int(time_parts[0]), int(time_parts[1]), 0)
minutes = int(input())
date = date + datetime.timedelta(minutes=minutes)
formatted_time = date.strftime("%H:%M")
print(formatted_time)